driverconfig/config.yaml

```diff
@@ -81,6 +81,7 @@ drivers:
       - configVersion: v2.6.0
         useDefaults: true
         supportedVersions:
+          - version: v121
```

Comment:
@VamsiSiddu-7 , please do the relevant changes so that K8s 1.21 will pick the latest version of sidecars